pstl/include/pstl/internal/algorithm_impl.h

```diff
@@ -1148,26 +1148,28 @@ __brick_count(_ForwardIterator __first, _ForwardIterator __last, _Predicate __pr
     return std::count_if(__first, __last, __pred);
 }
 
-template <class _ExecutionPolicy, class _ForwardIterator, class _Predicate, class _IsVector>
+template <class _Tag, class _ExecutionPolicy, class _ForwardIterator, class _Predicate>
 typename std::iterator_traits<_ForwardIterator>::difference_type
-__pattern_count(_ExecutionPolicy&&, _ForwardIterator __first, _ForwardIterator __last, _Predicate __pred,
-                /* is_parallel */ std::false_type, _IsVector __is_vector) noexcept
+__pattern_count(_Tag, _ExecutionPolicy&&, _ForwardIterator __first, _ForwardIterator __last, _Predicate __pred) noexcept
 {
-    return __internal::__brick_count(__first, __last, __pred, __is_vector);
+    return __internal::__brick_count(__first, __last, __pred, typename _Tag::_IsVector{});
```

Comment:
Should be:
```suggestion
    return __internal::__brick_count(__first, __last, __pred, typename _Tag::__is_vector{});
```
here and everywhere else. Otherwise, compilation error